wrappers/hnysqlx/sqlx.go

```diff
@@ -898,7 +904,7 @@ func (db *DB) SetMaxOpenConns(n int) {
 
 func (db *DB) Stats() sql.DBStats {
 	var err error
-	_, sender := common.BuildDBEvent(db.Builder, "")
+	_, sender := common.BuildDBEvent(db.Builder, db.Stats(), "")
```

Comment:
This looks to me like an endless loop.

Reply:
Ah, whoops, good catch. Yeah, we don't need to add this here.

Reply:
Do you think we even should be creating an event for this call? It can't even return an error, it's super fast, and if I'm adding it to every database call that will immediately double the number of database-related events emitted per DB call.

Reply:
I am on board to skip making a span for this call. Doubling the number of spans just to say "no really, every DB call also calls `Stats`" seems pointless.